textattack/shared/scripts/run_attack_parallel.py

```diff
@@ -94,6 +113,16 @@ def run(args):
         else:
             label, text = next(dataset)
             in_queue.put((label, text))
+
+        if args.checkpoint_interval and num_results % args.checkpoint_interval == 0:
+            chkpt_time = time.time()
+            date_time = datetime.datetime.fromtimestamp(chkpt_time).strftime('%Y-%m-%d %H:%M:%S')
+            print('\n\n' + '=' * 100)
+            logger.info('Saving checkpoint at {} after {} attacks.'.format(date_time, num_results))
+            print('=' * 100 + '\n')
+            checkpoint = textattack.shared.CheckPoint(chkpt_time, args, attack_log_manager)
+            checkpoint.save()
```

Comment:
I see this code repeated here and in `run_attack_single_threaded`. Maybe you could add the `CheckPoint` (now `Checkpoint`) class somehow? Then you could call 

``` 
checkpoint = ...
checkpoint.save(quiet=False)
```
and it'd automatically print that stuff. just an idea.

Reply:
I wrote the prints msgs before coming up with `Checkpoint`. Good idea